os-server/src/main/java/app/openschool/course/Course.java

```diff
@@ -90,6 +90,10 @@ public Course(String title, String description, Category category) {
     this.category = category;
   }
 
+  public static Course getInstance() {
```

Comment:
why do we need this method ?

Reply:
I've just used it in tests instead of an empty constructor.